### ISSUE TITLE
Example site 

### ISSUE THREAD

**Issue by stuartcampbell** [sciapp/sampledb#10]
It would be nice to be able to either see an example instance running (I see you link off to https://iffsamples.fz-juelich.de/instruments for example, but this isn't accessible to someone looking to test SampleDB.) or maybe give the option to populate with some test data.

I found myself with a few times with an entry form and not sure what information would be useful to put in the boxes.   This is where a populated example could really help.

**Comment by FlorianRhiem** [comment]
Hey @stuartcampbell,

thanks for the suggestion, currently demo.py is used to try out and demo features and changes in a prepopulated development environment, though the data there is parts realistic, parts for trying specific features only. It should be possible to set up something similar to that online with more realistic data, though I will have to see about how users will be able to sign in, making sure no emails are sent out, that the data is correctly reset periodically etc.

Probably the best route would be to first allow some way to setup a "demo" instance with the docker image, and then making it safe as an openly usable online demo next. I cannot promise that an online demo like this will be available soon, but it definitely sounds like something useful to showcase SampleDB and to provide some more assistance for new SampleDB admins.

**Comment by stuartcampbell** [comment]
I think something like a demo docker container, even if its a separate image that is run just to showcase what SampleDB can do.  

Thank you for considering the suggestion.

**Comment by FlorianRhiem** [comment]
True, something like demo.py for the docker image doesn't have the all the risks of an open demo on the web attached to it, so it should be much easier to achieve. I'll try and set up something like that next week. Thanks!

**Comment by FlorianRhiem** [comment]
There's now a new script for setting up demo data in the development version. After starting the sampledb container, it can be e.g. be called like this: `docker exec sampledb env/bin/python -m sampledb set_up_demo`

Once the next release is out (when this script is therefore included in the release Docker image), I will add that step to the guide as an optional command for setting up the demo data.
